src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -77,15 +86,14 @@ export class StudyViewPageStore {
 
     @observable private _chartVisibility = observable.map<boolean>();
 
-    private _clinicalAttributesMetaSet: { [id: string]: ChartMeta } = {} as any;
-
+    private _survivalClinicalData = observable.map<ClinicalData[]>();
```

Comment:
I'm pretty sure that obsevable.map is, by default, deeply observable.  Meaning, it passes values through observable.  If values are huge arrays of clinicalData (?) then this will be very costly, and probably unnecessary because the values don't change.  So use deep:false.